src/Core/Form/ChoiceProvider/CurrencyByIdChoiceProvider.php

```diff
@@ -63,4 +64,21 @@ public function getChoices()
 
         return $choices;
     }
+
+    public function getChoicesAttributes()
+    {
+        $currencies = $this->getCurrencies();
+        $choicesAttributes = [];
+
+        foreach ($currencies as $currency) {
+            $choicesAttributes[sprintf('%s (%s)', $currency['name'], $currency['iso_code'])]['symbol'] = $currency['symbol'];
```

Comment:
This code is a bit hard to read I suggest splitting in two lines:

```suggestion
            $currencyLabel = sprintf('%s (%s)', $currency['name'], $currency['iso_code']);
            $choicesAttributes[$currencyLabel]['symbol'] = $currency['symbol'];
```